system/Model.php

```diff
@@ -348,6 +348,42 @@ public function find($id = null)
 		return $row['data'];
 	}
 
+	/**
+	 * Fetches the column of database from $this->table with a primary key
+	 * matching $id.
+	 *
+	 * @param mixed|array|null $id One primary key or an array of primary keys
+	 * @param mixed|array|null $column_name Column name or null for primaryKey
+	 *
+	 * @return array|null array of table's column values
+	 */
+	public function findColumn($id = null, string $column_name = null) : array
```

Comment:
variables in CI4 should be camelCased, so `$columnName`

Reply:
Will change it in a moment. 
However I found some snake case variables as a function parameters:
 string $end_char = '&#8230;' $end_char $curl_options... so it is not consistent yet :).